images/cluster-setup/install.sh

```diff
@@ -45,92 +83,74 @@ get_clusters() {
     contexts=()
     kubeconfigs=()
     printf "Extracting files under the kubeconfig dir and reading the content in each file \n"
-    files=("$(ls "$DATA_DIR/gitops/sre/credentials/kubeconfig/compute")")
+    files=("$(ls "$WORKSPACE_DIR/credentials/kubeconfig/compute")")
     for kubeconfig in "${files[@]}"; do
-        printf "%s\n" "$kubeconfig"
-        subs=("$(KUBECONFIG=${DATA_DIR}/gitops/sre/credentials/kubeconfig/compute/${kubeconfig} kubectl config view -o jsonpath='{range .contexts[*]}{.name}{","}{.context.cluster}{"\n"}{end}')")
+        printf "  - %s\n" "$kubeconfig"
+        subs=("$(KUBECONFIG=${WORKSPACE_DIR}/credentials/kubeconfig/compute/${kubeconfig} kubectl config view -o jsonpath='{range .contexts[*]}{.name}{","}{.context.cluster}{"\n"}{end}')")
         for sub in "${subs[@]}"; do
             context=$(echo -n "${sub}" | cut -d ',' -f 1)
             cluster=$(echo -n "${sub}" | cut -d ',' -f 2 | cut -d ':' -f 1)
-	    if ! (echo "${clusters[@]}" | grep "${cluster}"); then
+            if ! (echo "${clusters[@]}" | grep "${cluster}"); then
                 clusters+=( "${cluster}" )
                 contexts+=( "${context}" )
                 kubeconfigs+=( "${kubeconfig}" )
-                printf "%s --- %s --- %s \n" "$cluster" "$context" "$kubeconfig"
+                printf "    - %s --- %s --- %s \n" "$cluster" "$context" "$kubeconfig"
             fi
         done
     done
+    printf "\n"
+}
 
+init_cluster_kubectl() {
```

Comment:
could be renamed:
switch_cluster()

rational:
- it is not an initialisation
- it makes clear what the intent is

Reply:
Fixed.